src/js/firebaseService.js

```diff
@@ -0,0 +1,117 @@
+let firebase = require('firebase/app');
```

Comment:
@Chem-e have you tried import statements here?

Reply:
yes

Reply:
@Chem-e still require statements 
please use import